STEERING-COMMITTEE.md

```diff
@@ -25,26 +25,26 @@ Given the nature of the discussions in Steering, meetings are not currently open
 
 ## Committee members
 
-Seats in Steering are held by a company, not by the individual users. Currently Steering consists of members from Google & IBM. These are:
+Seats in Steering are held by a company, not by the individual users. Currently Steering consists of members from Google, IBM & Red Hat. These are:
```

Comment:
seems like you removed the only red hat member though?

Reply:
I sorted him alphabetically.